docs/SUMMARY.md

```diff
@@ -20,16 +20,16 @@
     * [Google Analytics](integrations/sources/googleanalytics.md)
     * [Google Sheets](integrations/sources/google-sheets.md)
     * [Hubspot](integrations/sources/hubspot.md)
+    * [Marketo](integrations/sources/marketo.md)
     * [MySQL](integrations/sources/mysql.md)
     * [Postgres](integrations/sources/postgres.md)
     * [Rest API](integrations/sources/rest-api.md)
     * [Salesforce](integrations/sources/salesforce.md)
     * [Stripe](integrations/sources/stripe.md)
   * [Destinations](integrations/destinations/README.md)
-    * [BigQuery](integrations/destinations/bigquery.md)
     * [Local CSV](integrations/destinations/local-csv.md)
+    * [BigQuery](integrations/destinations/bigquery.md)
     * [Postgres](integrations/destinations/postgres.md)
-    * [Snowflake](integrations/destinations/snowflake.md)
```

Comment:
why is that disapearing?

Reply:
i think this was some gitbook weirdness. I didn't remove it from the gitbook UI. reset this branch to remove all other changes.